lang/en.yml

```diff
@@ -230,6 +230,9 @@ en:
     EMAILDETAILSTAB: 'Email Details'
     PLURALNAME: 'User Defined Form Email Recipients'
     SINGULARNAME: 'User Defined Form Email Recipient'
+    EMAILADDRESSINVALID: 'Email address has a invalid email address'
+    EMAILFROMINVALID: 'Email from has a invalid email address'
+    EMAILREPLYTOINVALID: 'Email reply has a invalid email address'
```

Comment:
This is minor, but it should be "has _an_ invalid email address". Would be best to fix this before it ends up being the source of truth in Transifex.